### ISSUE TITLE
UITooltip doesn't work inside some VueComponents

### ISSUE THREAD

**Issue by mcmillenb** [JosephusPaye/Keen-UI#204]
I tried to add a `<ui-tooltip>` to a `<ui-button>` like so:

```
<ui-button ref="edit-btn">
  Edit
  <ui-tooltip trigger="edit-btn">
    Select an element to edit
  </ui-tooltip>
</ui-button>
```

And get an error:

> Tooltip Error: You must provide a target for Tooltip to attach to

The problem is in UiTooltip.vue:

The `initialize()` method creates a new `Tooltip` object, passing `this.$parent.$refs[this.trigger],` as the target. However, the UiButton template wraps the content of the button in a div, so `this.$parent` is that wrapper div, and not the root element containing the `ref` value.

This is my workaround for now: 

```
<div ref="edit-btn">
  <ui-button>
    Edit
  </ui-button>
  <ui-tooltip trigger="edit-btn">
    Select an element to edit
  </ui-tooltip>
</div>
```

**Comment by EmilMoe** [comment]
This is how it's supposed to work to my understanding. You don't wrap a tooltip inside a button, the button just activates it.

**Comment by mcmillenb** [comment]
No, I don't think so. Unless you mean tooltips for `ui-button` components specifically are handled differently?

`ui-tooltip` components affect their parent element. See the [docs source code](https://github.com/JosephusPaye/Keen-UI/blob/master/docs-src/pages/UiTooltip.vue): 

```
<div class="image-pane" ref="image" :style="{ 'background-image': 'url(' + image + ')' }">
    <ui-tooltip trigger="image" :position="tooltipPosition">{{ name }}</ui-tooltip>
</div>
```

**Comment by lavezzi1** [comment]
UiTooltip doesn't work inside Ui-Tabs

**Comment by zdpeitang** [comment]
same issue, even workaround doesn't work for me.

**Comment by lavezzi1** [comment]
Any workaround or solution for that issue mr. @JosephusPaye ?

**Comment by JosephusPaye** [comment]
This has been fixed in https://github.com/JosephusPaye/Keen-UI/commit/d8edc3179a01796287524926505dd14bc4278c00.

The [workaround ](https://github.com/JosephusPaye/Keen-UI/issues/204#issue-205021308)@mcmillenb posted can be used until the next version.

**Comment by JosephusPaye** [comment]
In the upcoming version, you can also use a CSS selector for the trigger:

```vue
<ui-button id="edit-btn">Edit</ui-button>
<ui-tooltip trigger="#edit-btn">Select an element to edit</ui-tooltip>
```

**Comment by JosephusPaye** [comment]
Released in [v1.1.0](https://github.com/JosephusPaye/Keen-UI/releases/tag/v1.1.0).
